Makefile

```diff
@@ -43,10 +43,10 @@ services:
 
 # these are useful for devs
 jobs:
-	docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
-	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
-	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest
-	docker run -it -v "${CURDIR}/ci-operator/jobs:/ci-operator/jobs" registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest --prow-jobs-dir /ci-operator/jobs
+	podman pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
```

Comment:
you should alias `docker` to `podman` on your machine -- this is a no-go for merging into this repo